ipn/ipnstate/ipnstate.go

```diff
@@ -81,6 +85,18 @@ type TailnetStatus struct {
 	MagicDNSEnabled bool
 }
 
+// ExitNodeStatus is the exit node used by the local backend.
```

Comment:
"local backend" is an internal term and these is somewhat user-facing docs.

Maybe just repeat some of the docs above?

// ExitNodeStatus describes the state of an exit node.